### PR TITLE
Theunis logout funtionality

### PR DESCRIPTION
**Pull request by TheunisK**

### CODE REVIEW

File: src/client/pages/signUp/SignUp.jsx [hunk 1]
```diff
@@ -109,7 +111,8 @@ const SignUp = (props) => {
 }
 
 SignUp.propTypes = {
-    setUserId: PropTypes.string.isRequired
+    setUserId: PropTypes.func.isRequired,
```
[the remark this answers is not in the document]
Comment (polarbear23) [reply src/client/pages/signUp/SignUp.jsx:114]: true I love living in this pr its so nice

[the remark this answers is not in the document]
Comment (nathanSamosa) [reply src/client/pages/signUp/SignUp.jsx:114]: Shut up bravin

[the remark this answers is not in the document]
Comment (TheunisK) [reply src/client/pages/signUp/SignUp.jsx:114]: Bravin pulling like its a Saturday Night in London...

File: src/client/App.jsx [hunk 2]
```diff
@@ -10,15 +10,22 @@ import MatchPage from './pages/matchPage/MatchPage';
 import MyRecommendations from './pages/myRecommendations/MyRecommendations';
 
 function App() {
+  const [loggedIn, setLoggedIn] = useState(false);
   const [userId, setUserId] = useState('');
 
+ /* dont think we need this useEffect line. can check in signIn and signUp if token is created then set loggedIn */
```
[the remark this answers is not in the document]
Comment (TheunisK) [reply src/client/App.jsx]: Done.

**Pull request merged**